Makefile

```diff
@@ -110,8 +110,11 @@ proto-format: check-proto-format-deps
 .PHONY: proto-format
 
 proto-check-breaking: check-proto-deps
-	@echo "Checking for breaking changes in Protobuf files"
-	@buf breaking
+	@echo "Checking for breaking changes in Protobuf files against local branch"
+	@echo "Note: This is only useful if your changes have not yet been committed."
+	@echo "      Otherwise read up on buf's \"breaking\" command usage:"
+	@echo "      https://docs.buf.build/breaking/usage"
+	@buf breaking --against ".git"
```

Comment:
You can make this against your master branch by specifying `--against .git#branch=master` https://docs.buf.build/reference/inputs#git

Reply:
We wouldn't necessarily want that - these Makefile commands are for local use only.

We'll have separate CI-based breaking change checks for release branches, where there should never be breaking changes, but we want changes on `master` to be able to be breaking.

Reply:
Sounds good - whatever works for you. We do it a different way (`main` always expected to be deployable), but everyone has their own method :-)